docs/Job-reference.md

```diff
@@ -5544,6 +5545,38 @@ job('example') {
 
 (since 1.26)
 
+### [Crittercism dSYM Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Crittercism+dSYM+Plugin)
+
+```groovy
+job {
+    publishers {
+        crittercismDsymUpload {
+            apiKey(String apiKey = '')
+            appID(String appID = '')
+            filePath(String filePath = '')
+        }
+    }
+}
+```
+
+Adds a Crittercism dSYM Upload publisher. Requires the [Crittercism dSYM Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Crittercism+dSYM+Plugin).
+
```

Comment:
Add a link to the new "Handling Credentials" page to give the user a hint about handling plain text credentials. See https://github.com/jenkinsci/job-dsl-plugin/blob/e917f45419ca2cbcf4cb95e673acdda6667eeeef/docs/Job-reference.md#hipchat-publisher for an example.